src/pages/MapPage.js

```diff
@@ -13,18 +13,18 @@ const MapPage = () => {
 
   return (
     <>
-      <h1>What's Near Me?</h1>
-      <div className="view__content">
+      <div className="view__header">
+        <h1>What's Near Me?</h1>
```

Comment:
@segdeha's comment made me wonder, should this `<h1>` be wrapped in a `<header>` for more semantic HTML?  I am not clear yet based on the semantic HTML and React docs that I've looked at so far. 

Definitely just a question, and not a block to merging!

Reply:
Yes, Jessica! Thank you! I think you are right. I'm going to change to a `<header>` tag.